rpcs3/Emu/RSX/GL/GLGSRender.cpp

```diff
@@ -335,9 +335,9 @@ void GLGSRender::on_init_thread()
 
 	if (!m_overlay_manager)
 	{
-		m_frame->hide();
+		show_window();
```

Comment:
Typo?

Reply:
Yes, can these lines be removed? hide() hides an already hidden window, show() shows an empty white window if I undertand correctly.

Reply:
I'm referring to https://github.com/RPCS3/rpcs3/pull/9633#discussion_r562134148 mainly

Reply:
You forgot to answer https://github.com/RPCS3/rpcs3/pull/9633#discussion_r562134182

Reply:
it depends on whether it's guaranteed that the game window is always shown afterwards and never shown beforehand.
The game window is being hidden because it may happen that the Qt dialog pops up behind the game window, especially in fullscreen mode.

Reply:
Fixed that empty white window case then.